env-files/tensorflow/generic_tf.sh

```diff
@@ -88,9 +88,9 @@ pip3 install --no-cache-dir  tf_keras==2.16.*
 
 # Install Pov4ML
 if [[ "$OSTYPE" =~ ^darwin ]] ; then
-  pip install "prov4ml[apple]@git+https://github.com/matbun/ProvML@main" || exit 1
+  pip install "prov4ml[apple,nvidia]@git+https://github.com/matbun/ProvML@new-main" || exit 1
 else
-  pip install "prov4ml[linux]@git+https://github.com/matbun/ProvML@main" || exit 1
+  pip install "prov4ml[linux,nvidia]@git+https://github.com/matbun/ProvML@new-main" || exit 1
```

Comment:
I might be mistaken here, but from what I can tell, the `new-main` branch doesn't have an extra called `linux` anymore. The `linux` in the `main` branch seems to contain what `amd` contains in the `new-main` branch.